.github/workflows/pythonpublish.yml

```diff
@@ -3,7 +3,7 @@
 
 name: Upload Python Package
 
-on:
+true:
```

Comment:
This shouldbe  `on:`.
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#on

Reply:
I changed it because of:

```
yamllint.................................................................Failed
- hook id: yamllint
- exit code: 1

.github/workflows/test.yml
  6:1       error    truthy value should be one of [false, true]  (truthy)
```

Reply:
If you want to keep linting the github actions yaml files, I'd adjust the linting settings to be compatible with the github actions syntax.

https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.truthy